GameFruitBasket/GameFruitBasket/Classes/Library.cs

```diff
@@ -0,0 +1,158 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace GameFruitBasket.Classes
+{
+    public static class Library
+    {
+        public static void AddTryToList(int number, List<int> tries)
+        {
+            tries.Add(number);
+        }
+
+        public static int GetTryNumber(out int number, int startRandom)
+        {
+            Random rnd = new Random(new Random().Next(5, startRandom));
+            number = rnd.Next(40, 140);
+            return number;
+        }
+
+        public static bool CheckList(int number, List<int> tries)
+        {
+            bool InMemory = false;
+            for (int i = 0; i < tries.Count; i++)
+            {
+                if (tries[i].Equals(number))
+                {
+                    InMemory = true;
+                }
+            }
+            return InMemory;
+        }
+
+        public static bool CheckListForCheater(int number)
+        {
+            bool InMemory = false;
+
+            for (int i = 0; i < AllLists.allLists.Count; i++)
+            {
+                for (int j = 1; j < AllLists.allLists[i].Count; j++)
+                {
+                    if (AllLists.allLists[i][j].Equals(number))
+                    {
+                        InMemory = true;
+                    }
+                }
+            }
+            return InMemory;
+        }
+
+        public static bool CheckListForGame(int number)
+        {
+            bool InMemory = false;
+            for (int r = 1; r < 7; r++)
+            {
+                int approxiBasketWeight1 = r + Basket.Weight;
+                int approxiBasketWeight2 = Basket.Weight - 1;
+                for (int i = 0; InMemory == false && i < AllLists.allLists.Count; i++)
+                {
+                   // Console.WriteLine($"{i + 1}List:");
+                    for (int j = 0; InMemory == false && j < AllLists.allLists[i].Count; j++)
+                    {
+                        //Console.WriteLine($"{j}el={AllLists.allLists[i][j]} сравниваем с {r} ");
+
+                        if (AllLists.allLists[i][j].Equals(approxiBasketWeight1) ||
+                            AllLists.allLists[i][j].Equals(approxiBasketWeight2))
+                        {
+                            PrintVictoryMessage(AllLists.allPlayers[i], "approximate");
+                            //Console.WriteLine("!!!!ну хоть приблизительно");  
+                            InMemory = true;
+                            break;
+                        }
+                        //else Console.WriteLine("Не равно");
+                    }
+                }
+            }
+
+            return InMemory;
+        }
+
+        public static void Game()
+        {
```

Comment:
According to naming convention methods name have to contain verb
e.g. "StartGame", "BeginGame"